lib/uvm/bridge-client.js

```diff
@@ -27,13 +37,13 @@ bridge = (function (emit, jsonStringify, arrayProtoSlice) {
             this._events[name].push(listener);
         }
     };
-}(__uvm_emit, JSON.stringify, Array.prototype.slice));
+}(__uvm_emit, CircularJSON.stringify, Array.prototype.slice));
 
 __uvm_dispatch = (function (bridge, bridgeEmit, jsonParse) {
     return function (args) {
         bridgeEmit.apply(bridge, jsonParse(args));
     };
-}(bridge, bridge.emit, JSON.parse));
-
+}(bridge, bridge.emit, CircularJSON.parse));
+delete CircularJSON;
```

Comment:
Does it delete the variable from the context? Tried it, doesn't seem so.
![screen shot 2016-12-19 at 2 04 05 pm](https://cloud.githubusercontent.com/assets/1330851/21305892/163f2d6c-c5f4-11e6-929d-3d2fe6df4fc8.png)